packages/settings-library/src/settings_library/tracing.py

```diff
@@ -1,21 +1,14 @@
-from pydantic import AnyUrl, Field, parse_obj_as
+from pydantic import AnyUrl, Field, conint
 
 from .base import BaseCustomSettings
 
 UNDEFINED_CLIENT_NAME = "undefined-tracing-client-name"
 
 
 class TracingSettings(BaseCustomSettings):
-    TRACING_ZIPKIN_ENDPOINT: AnyUrl = Field(
-        default=parse_obj_as(AnyUrl, "http://jaeger:9411"),
-        description="Zipkin compatible endpoint",
+    TRACING_OTEL_COLLECTOR_ENDPOINT: AnyUrl | None = Field(
+        description="Opentelemetry compatible collector endpoint"
     )
-    TRACING_THRIFT_COMPACT_ENDPOINT: AnyUrl = Field(
-        default=parse_obj_as(AnyUrl, "http://jaeger:5775"),
-        description="accept zipkin.thrift over compact thrift protocol (deprecated, used by legacy clients only)",
-    )
-    TRACING_CLIENT_NAME: str = Field(
-        default=UNDEFINED_CLIENT_NAME,
-        description="Name of the application connecting the tracing service",
-        env=["HOST", "HOSTNAME", "TRACING_CLIENT_NAME"],
+    TRACING_OTEL_COLLECTOR_PORT: conint(ge=1024, le=65535) | None = Field(  # type: ignore
+        description="Opentelemetry compatible collector port"
```

Comment:
Question: Why are these 2 entries Optional? What does that mean? I can define TracingSettings without ENDPOINT?? without Port would there be a default one instead of None?
Also mypy is right to say this is wrong.

Reply:
There is a type  https://github.com/itisfoundation/osparc-simcore/blob/5884e48f7c845f5050942fe0be32dad2bc592139/packages/settings-library/src/settings_library/basic_types.py#L12-L15

Q: why this port >1024? If this is a known port range perhaps you can create a new port constraint in basic_types?

Reply:
MINOR: I do not think these descriptions add any extra info since the name is quite self explanatory. I would rather add something that is not in not already stated in the name.